client/cacher/finder.go

```diff
@@ -0,0 +1,63 @@
+package cacher
+
+import (
+	"context"
+	"encoding/json"
+	"net"
+
+	cacherClient "github.com/packethost/cacher/client"
+	"github.com/packethost/cacher/protos/cacher"
+	"github.com/pkg/errors"
+	"github.com/tinkerbell/boots/client"
+)
+
+type CacherFinder struct {
```

Comment:
Might not need to include `Cacher` in names in this file with the package being called `cacher`.
prefer `cacher.Finder` over `cacher.CacherFinder`

_"All references to names in your package will be done using the package name, so you can omit that name from the identifiers."_ --[ref](https://github.com/golang/go/wiki/CodeReviewComments#package-names)

Reply:
Fixed!